src/raft.c

```diff
@@ -599,6 +637,40 @@ static void clearClientSessions(RedisRaftCtx *rr)
     rr->client_session_dict = RedisModule_CreateDict(rr->ctx);
 }
 
+static void timeoutBlockedCommand(RedisRaftCtx *rr, raft_entry_t *entry)
+{
+    RedisModule_Assert(entry->data_len == sizeof(raft_index_t));
+    raft_index_t idx;
+    memcpy(&idx, entry->data, sizeof(idx));
```

Comment:
```suggestion
    RedisModule_Assert(entry->data_len == sizeof(raft_index_t));

    raft_index_t idx;
    memcpy(&idx, entry->data, sizeof(idx));
```
memcpy looks like a variable to my eyes without a single empty line

Reply:
ok, I wanted them next to each other as they go together, its basically idx being initalized.  perhaps one could say something like 

`raft_idx_t idx = *(raft_idx_t *) entry->data`

will that (or something like it) work.

or perhaps just 

`raft_index_t *idx = entry->data`

and then just use `*idx` elsewhere in the code?

Reply:
ok, changing this to be relative to your comment re serialized below.  I'm not going to get the multibulk right, so could use feedback on that.